packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -473,6 +473,7 @@ class SemanticsNode extends AbstractNode {
     owner._nodes.remove(id);
```

Comment:
@abarth changes in this file are new since your review. they fix a bug we've seen people complain about which the gallery test started hitting (presumably because it's now lazy -- it's about when you move nodes out of the tree with semantics on)